examples/js_of_ocaml/.jscsrc

```diff
@@ -0,0 +1,69 @@
+{
+	"fileExtensions": [".js"],
+	"requireCurlyBraces": [
+		"if",
+		"else",
+		"for",
+		"while",
+		"do",
+		"try",
+		"catch"
+	],
+	"requireSpaceAfterKeywords": [
+		"if",
+		"else",
+		"for",
+		"while",
+		"do",
+		"switch",
+		"case",
+		"return",
+		"try",
+		"catch",
+		"function",
+		"typeof"
+	],
+	"excludeFiles": ["**/node_modules/**", "**/bower_components/**", "**/js/todomvc.js"],
+	"requireSpaceBeforeBlockStatements": true,
+	"requireParenthesesAroundIIFE": true,
+	"requireSpacesInConditionalExpression": true,
+	"disallowSpacesInNamedFunctionExpression": {
+		"beforeOpeningRoundBrace": true
+	},
+	"disallowSpacesInFunctionDeclaration": {
+		"beforeOpeningRoundBrace": true
+	},
+	"disallowMultipleVarDecl": "exceptUndefined",
+	"requireBlocksOnNewline": 1,
+	"disallowEmptyBlocks": true,
+	"disallowSpacesInsideArrayBrackets": true,
+	"disallowSpacesInsideParentheses": true,
+	"disallowQuotedKeysInObjects": true,
+	"disallowSpaceAfterObjectKeys": true,
+	"requireCommaBeforeLineBreak": true,
+	"disallowSpaceAfterPrefixUnaryOperators": true,
+	"disallowSpaceBeforePostfixUnaryOperators": true,
+	"disallowSpaceBeforeBinaryOperators": [
+		","
+	],
+	"requireSpaceBeforeBinaryOperators": true,
+	"requireSpaceAfterBinaryOperators": true,
+	"requireCamelCaseOrUpperCaseIdentifiers": true,
+	"validateQuoteMarks": "'",
+	"disallowMixedSpacesAndTabs": true,
+	"disallowTrailingWhitespace": true,
+	"disallowTrailingComma": true,
+	"disallowKeywordsOnNewLine": [ "else" ],
+	"requireLineFeedAtFileEnd": true,
+	"requireCapitalizedConstructors": true,
+	"requireDotNotation": true,
+	"disallowYodaConditions": true,
+	"disallowNewlineBeforeBlockStatements": true,
+	"maximumLineLength": 120,
+	"validateLineBreaks": "LF",
+	"validateIndentation": "\t",
+	"disallowKeywords": ["with"],
+	"disallowSpacesInsideObjectBrackets": null,
+	"disallowImplicitTypeConversion": ["string"],
+	"safeContextKeyword": "self"
+}
```

Comment:
If it's possible to propose a modification of the `.jscsrc` file (https://github.com/tastejs/todomvc/blob/master/.jscsrc) then maybe it could be something more generic? Something like:

```
"excludeFiles": ["**/node_modules/**", "**/bower_components/**", "**/auto_gen/**"],
```

So if an implementation uses some automatically generated js files, it can put them in this directory to prevent the `jscsrc` checkings.